docs/Administration.rst

```diff
@@ -2366,14 +2393,44 @@ Managing memberships
 Select the Memberships tab. Here, you can manage user roles in a
 specific group.
 
-By default, 9 membership types are available in PRODUCT including
-Member, Author, Editor, Manager, Redactor, Validator, Webdesigner,
-Publisher and \*.
+By default, 9 membership types are available in eXo Platform including
+Member, Author, Editor, Manager, Redactor, Validator, Webdesigner, Publisher and \*.
 
 |image178|
 
+The membership, such as `manager:/platform/users`, is defined by both membership type (manager, member, author...) 
+and a group i.e. it is the combinaison between a group and a membership type to define the user's role in the group.
+
 .. note:: The \* membership is identical to any other type, so choosing this type means adding all available membership types.
 
+**Membership usage in eXo Platform**
+
+In eXo Platform, the most used memships are `manager`, `member` and `*`.
+
+- The `manager` role to define users with managing privileges
```

Comment:
I think it gives space management peivileges but I also thnk manager membership type has something to do with navigation management in the prtal and maybe administrative privileges in community management (to ne verified, I may be wrong)